packages/node_modules/@node-red/editor-api/lib/admin/flows.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
             if (version === "v1") {
                 opts.flows = {flows: req.body}
             } else {
-                opts.flows = req.body;
+                opts.flows = body.flows;
```

Comment:
Please revert this change as well. `body.flows` is not defined - I assume this is left over from the previous change that was reverted.

Reply:
I am sorry.  Reverted the code.